docs/lro-provisioning-state-specified.md

```diff
@@ -14,12 +14,15 @@ ARM OpenAPI(swagger) specs
 
 ## Output Message
 
-`${code} response in long running ${verb} operation is missing ProvisioningState property. A LRO PUT and PATCH operations response schema must have ProvisioningState specified.
+${code} response schema in long running PATCH operation is missing ProvisioningState property. A LRO PATCH operations 200 response schema must have ProvisioningState specified.
+${code} response schema in long running PUT operation is missing ProvisioningState property. A LRO PUT operations response schema must have ProvisioningState specified.
+200 response schema in long running DELETE operation is missing ProvisioningState property. A LRO DELETE operations 200 response schema must have ProvisioningState specified.
 
```

Comment:
You can combine the statements and state

Response schema in long running PUT, PATCH or DELETE operation is missing ProvisioningState property. A LRO PATCH operations 200 response schema must have ProvisioningState specified.